src/Compilers/Server/VBCSCompilerTests/VBCSCompilerTests.csproj

```diff
@@ -25,7 +25,7 @@
       <Aliases>vbc</Aliases>
     </ProjectReference>
     <ProjectReference Include="..\..\..\Test\PdbUtilities\PdbUtilities.csproj" />
-    <ProjectReference Include="..\..\..\Test\Utilities\Desktop\TestUtilities.Desktop.csproj" />
+    <ProjectReference Include="..\..\..\Test\Utilities\Portable\TestUtilities.csproj" />
```

Comment:
This required `Traits.cs` to move to Portable. I'm not sure what the contents of Traits.cs are used for - so I'm guessing this change (to reference Portable instead) is incorrect, but I'm not sure what else to do.